doonut-app-external-api/src/main/kotlin/com/doonutmate/calendar/controller/dto/CalendarResponse.kt

```diff
@@ -0,0 +1,17 @@
+package com.doonutmate.calendar.controller.dto
+
+import com.doonutmate.doonut.member.model.ProfileImage
+import io.swagger.v3.oas.annotations.media.Schema
+import java.time.Instant
+
+@Schema(title = "캘린더 조회 응답")
+data class CalendarResponse(
+    val id: Long,
+    val totalCount: Int,
+    val memberName: String,
+    val calendarName: String,
+    val updatedAt: Instant,
+    val firstUploadedAt: Instant,
+    val lastUploadedAt: Instant,
+    val imageUrl: List<ProfileImage>
```

Comment:
서버 단에선 ProfileImage 이미지가 List이지만 프론트엔드에선 대표이미지 1개만 필요하기 때문에 혼란스러울 수 있어요.  
imageType이 REPRESENTATIVE인 이미지 1개만 내리는게 좋을 것 같습니다!

```suggestion
    val profileImageUrl: String
```